bokehjs/src/lib/document/document.ts

```diff
@@ -248,13 +251,15 @@ export class Document implements Equatable {
     return this._roots
   }
 
-  protected _add_root(model: HasProps): boolean {
-    if (includes(this._roots, model))
+  protected _add_root(...models: HasProps[]): boolean {
```

Comment:
since this is private why not pluralize to `_add_roots` to be more descriptive of the new signature